crates/pixi_build_types/src/procedures/conda_metadata.rs

```diff
@@ -1,3 +1,5 @@
+//! This API was introduced in Pixi Build API version 0.
```

Comment:
this will be `deprecated` in a way that we will not use conda-getmetadata but conda-outputs?